flutter/pubspec.lock

```diff
@@ -1547,5 +1547,5 @@ packages:
     source: hosted
     version: "0.1.1"
 sdks:
-  dart: ">=2.18.0 <4.0.0"
+  dart: ">=2.18.0 <3.0.0"
```

Comment:
Why?

Reply:
Flutter's [latest release](https://docs.flutter.dev/get-started/install/linux) contains flutter@3.7.3 and dart@2.19.2

In [pubspec.yaml](https://github.com/rustdesk/rustdesk/blob/master/flutter/pubspec.yaml) file there's no upperbound for dart, refer to `environment.sdk` value. Therefore while resolving dependencies, flutter automatically created the upperbound in `pubspec.lock` file to `3.0.0` for dart i.e at `2.19.2`.